#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_x-terminal-emulator - Mattia Dongili <malattia@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use Debian's x-terminal-emulator

@DPATCH@

# Author: malattia
# Status: none

--- arch/um/drivers/xterm.c.orig        2002-10-04 23:51:39.000000000 -0400
+++ arch/um/drivers/xterm.c     2002-10-04 23:55:14.000000000 -0400
@@ -45,7 +45,7 @@
 }
 
 /* Only changed by xterm_setup, which is a setup */
-static char *terminal_emulator = "xterm";
+static char *terminal_emulator = "x-terminal-emulator";
 static char *title_switch = "-T";
 static char *exec_switch = "-e";
 
@@ -76,7 +76,7 @@
 "    respectively.  The title switch must have the form '<switch> title',\n"
 "    not '<switch>=title'.  Similarly, the exec switch must have the form\n"
 "    '<switch> command arg1 arg2 ...'.\n"
-"    The default values are 'xterm=xterm,-T,-e'.  Values for gnome-terminal\n"
+"    The default values are 'xterm=x-terminal-emulator,-T,-e'.  Values for gnome-terminal\n"
 "    are 'xterm=gnome-terminal,-t,-x'.\n\n"
 );
 
